sky/shell/platform/mojo/dart_tracing.h

```diff
@@ -0,0 +1,61 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SKY_SHELL_PLATFORM_MOJO_DART_TRACING_H_
+#define SKY_SHELL_PLATFORM_MOJO_DART_TRACING_H_
+
+#include "base/trace_event/trace_event.h"
+#include "mojo/common/tracing_impl.h"
+#include "mojo/public/cpp/bindings/interface_request.h"
+#include "mojo/services/tracing/interfaces/tracing.mojom.h"
+
+namespace dart {
+
+class DartTimelineController {
```

Comment:
Done. Also made all these static methods just global functions in anonymous namespace and dropped DartTimelineController.